foundry/tensorflow_dataset_wrapper.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+from tensorflow.keras.utils import Sequence
+
+class FoundryDatasetAsTensorflowSequence(Sequence):
+    """Foundry Dataset Converted to Tensorflow Format"""
+
+    def __init__(self, inputs, targets):
```

Comment:
You should document what the inputs to these classes are.

Reply:
really anything. all foundry datasets have data separated into inputs and targets. so inputs can really be anything, and that has to be the case since it would be ridiculously hard to accommodate for every dataset otherwise.